File: MyApplication/app/src/main/java/com/example/lloc/ecologic/Question.java
package com.example.lloc.ecologic;

/**
 * Created by lLoïc on 05/11/2017.
 */

public class Question {
    private String intitule,rep1,rep2,rep3,rep4,bonnereponse,explication;
    private int id,score;

    public Question(){}

    public Question (int id,String intitule,String rep1, String rep2, String rep3, String rep4, String bonnereponse,String explication,int score)
    {
        this.id = id;
        this.intitule = intitule;
        this.rep1 = rep1;
        this.rep2 = rep2;
        this.rep3 = rep3;
        this.rep4 = rep4;
        this.bonnereponse = bonnereponse;
        this.explication=explication;
        this.score =score;
    }

    public int getScore() {return score; }
    public String getExplication() {return explication; }
    public String getIntitule() {
        return intitule;
    }

    public void setIntitule(String intitule) {
        this.intitule = intitule;
    }

    public String getReponse() {
        return bonnereponse;
    }

    public void setReponse(String reponse) {
        this.bonnereponse = reponse;
    }

    public String getRep4() {

        return rep4;
    }

    public void setRep4(String rep4) {
        this.rep4 = rep4;
    }

    public String getRep3() {

        return rep3;
    }

    public void setRep3(String rep3) {
        this.rep3 = rep3;
    }

    public String getRep2() {

        return rep2;
    }

    public void setRep2(String rep2) {
        this.rep2 = rep2;
    }

    public String getRep1() {

        return rep1;
    }

    public void setRep1(String rep1) {
        this.rep1 = rep1;
    }

    public String verifierReponse(String repUtilisateur){
        return (this.bonnereponse == repUtilisateur ? "Bonne Réponse" : "Mauvaise Réponse");
    }

}

File: web/backoffice/entities/Question.php
<?php

class Question {
  private $_id,$_intitule,$_rep1,$_rep2,$_rep3,$_rep4,$_goodrep,$_explication,$_score,$_theme;

  public function __construct($_id,$_score,$_theme,$_intitule,$_rep1,$_rep2,$_rep3,$_rep4,$_goodrep,$_explication) {
    $this->_id = $_id;
    $this->_score = $_score;
    $this->_theme = $_theme;
    $this->_intitule = $_intitule;
    $this->_rep1 = $_rep1;
    $this->_rep2 = $_rep2;
    $this->_rep3 = $_rep3;
    $this->_rep4 = $_rep4;
    $this->_goodrep = $_goodrep;
    $this->_explication = $_explication;
  }


  public function getTheme() {
    return $this->_theme;

  }
  
  public function getId() {
    return $this->_id;
  }

    /**
     * @return mixed
     */
    public function getRep4()
    {
        return $this->_rep4;
    }

    /**
     * @return mixed
     */
    public function getGoodrep()
    {
        return $this->_goodrep;
    }

    /**
     * @return mixed
     */
    public function getExplication()
    {
        return $this->_explication;
    }

    /**
     * @return mixed
     */
    public function getScore()
    {
        return $this->_score;
    }

  public function getIntitule() {
    return $this->_intitule;
  }

  public function getRep1() {
    return $this->_rep1;
  }

  public function getRep2() {
    return $this->_rep2;
  }

  public function getRep3() {
    return $this->_rep3;
  }
}
